distributed/worker.py

```diff
@@ -1381,28 +1398,32 @@ def transition_dep_waiting_flight(self, dep, worker=None):
                 pdb.set_trace()
             raise
 
-    def transition_dep_flight_waiting(self, dep, worker=None):
+    def transition_dep_flight_waiting(self, dep, worker=None, busy=False):
```

Comment:
This keyword should be inverted and changed to something like `remove`, which probably makes more sense locally